### PR TITLE
Add Bootstrap 3.3.6 and JQuery 2.2.3 resources

### PR DESCRIPTION
**Pull request by urlgrey**
Update Bootstrap to 3.3.6 (latest stable) and JQuery 2.2.3 (latest stable).  This will make it easier to maintain going forward, and adds a fresh look to the UI.

### CODE REVIEW

**Comment by argilo** [comment]
There's a bunch of `span*` that need to be changed to `col-md-*` in the following files:

* View/NetworkServices/index.ctp
* View/Status/index.ctp
* View/WifiScan/index.ctp

**Comment by argilo** [comment]
In View/NetworkServices/index.ctp, `array('class' => 'icon-trash'));` needs to change to `array('class' => 'glyphicon glyphicon-trash'));`

File: src/var/www/hsmm-pi/View/Layouts/default.ctp [hunk 1]
```diff
@@ -28,175 +28,179 @@ $cakeDescription = __d('cake_dev', __('HSMM-Pi'));
...
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+            <h3 id="modalLabelReboot"><?php echo __('Reboot Confirmation'); ?></h3>
+          </div>
+          <div class="modal-body">
+            <p>The node will be unavailable during the reboot.  Are you sure you want to do this?</p>
+          </div>
+          <div class="modal-footer">
+            <button class="btn" data-dismiss="modal" aria-hidden="true">Close</button>
```
Comment (argilo) [inline src/var/www/hsmm-pi/View/Layouts/default.ctp]: `class="btn btn-default"`

File: src/var/www/hsmm-pi/View/Layouts/default.ctp [hunk 2]
```diff
@@ -28,175 +28,179 @@ $cakeDescription = __d('cake_dev', __('HSMM-Pi'));
...
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+            <h3 id="modalLabelShutdown"><?php echo __('Shutdown Confirmation'); ?></h3>
+          </div>
+          <div class="modal-body">
+            <p>The node will be totally unavailable after shutdown.  Are you sure you want to do this?</p>
+          </div>
+          <div class="modal-footer">
+            <button class="btn" data-dismiss="modal" aria-hidden="true">Close</button>
```
Comment (argilo) [inline src/var/www/hsmm-pi/View/Layouts/default.ctp]: `class="btn btn-default"`

File: src/var/www/hsmm-pi/View/Status/index.ctp [hunk 3]
```diff
@@ -177,22 +177,26 @@ foreach ($mesh_services as $service) {
...
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+        <h3 id="modalLabelMap">Node Location Map</h3>
+      </div>
+      <div class="modal-body">
+        <div id='mapDiv' style="position:relative; width:500px; height:350px;"></div>
+        <h5>Latitude:&nbsp;<em id="latitude"></em>&nbsp;&nbsp;Longitude:&nbsp;<em id="longitude"></em></h5>
+      </div>
+      <div class="modal-footer">
+        <button class="btn" data-dismiss="modal" aria-hidden="true">Close</button>
```
Comment (argilo) [inline src/var/www/hsmm-pi/View/Status/index.ctp]: `class="btn btn-default"`

**Comment by argilo** [comment]
And one more `class="btn btn-default"` is needed in View/NetworkServices/add.ctp.

**Comment by argilo** [comment]
`alert-error` needs to change to `alert-danger` in Elements/Flash/error.ctp and Status/index.ctp.

**Comment by argilo** [comment]
There's an extra `</div>` somewhere, I'll let you know if I track it down.

**Comment by urlgrey** [comment]
Thanks, there's a bit more work involved in the form layout as well.  I'll keep on it.  Thanks for the prompt feedback!

File: src/var/www/hsmm-pi/View/Layouts/default.ctp [hunk 4]
```diff
@@ -28,175 +28,179 @@ $cakeDescription = __d('cake_dev', __('HSMM-Pi'));
...
+            ?>
+          </li>
+        </ul>
         <?php
           }
         ?>
-        </div>
       </div>
     </div>
   </div>
```
Comment (argilo) [inline src/var/www/hsmm-pi/View/Layouts/default.ctp]: Looks like this `</div>` should go.

Comment (urlgrey) [inline src/var/www/hsmm-pi/View/Layouts/default.ctp]: Yep, that'll match up the divs

File: src/var/www/hsmm-pi/View/NetworkServices/index.ctp [hunk 5]
```diff
@@ -49,7 +49,7 @@ echo $this->Html->link('', array(
 			'action' => 'delete',
 			$service['NetworkService']['id'],
 		),
-			array('class' => 'icon-trash'));
+			array('class' => 'glyphicon glyphicon-trash'));
 		?>
           </td>
 	  </div></td>
```
Comment (argilo) [inline src/var/www/hsmm-pi/View/NetworkServices/index.ctp]: Another extra `</div>` to get rid of.

Comment (argilo) [inline src/var/www/hsmm-pi/View/NetworkServices/index.ctp]: And looks like the one on line 69 should be deleted too.

Comment (argilo) [inline src/var/www/hsmm-pi/View/NetworkServices/index.ctp]: Also the `</td>` on line 55 should go away.

File: src/var/www/hsmm-pi/View/NetworkServices/index.ctp [hunk 6]
```diff
@@ -4,7 +4,7 @@
 </div>
 
 <div class="row">
-  <div class="span12">
+  <div class="col-md-12">
     <span class="pull-right">
```
Comment (argilo) [inline src/var/www/hsmm-pi/View/NetworkServices/index.ctp]: `<p>` is not allowed inside `<span>`, so change this to a `<div>` instead?

File: src/var/www/hsmm-pi/View/LocationSettings/edit.ctp [hunk 7]
```diff
@@ -17,6 +17,11 @@
 
 <?php
 echo $this->Form->create('LocationSetting', array(
+  'inputDefaults' => array(
+    'div' => 'form-group',
+    'label' => array('class' => 'col col-md-3 control-label'),
+    'wrapInput' => 'col col-md-9',
```
Comment (argilo) [inline src/var/www/hsmm-pi/View/LocationSettings/edit.ctp]: These `wrapInput`s are making it into the HTML as attributes on `<input>` tags, I don't imagine you want that. The option seems to be intended for a plugin you're not using.

Comment (urlgrey) [inline src/var/www/hsmm-pi/View/LocationSettings/edit.ctp]: You're right; I'll remove the `wrapInput`s, as they're having no effect.

File: src/var/www/hsmm-pi/View/NetworkServices/index.ctp [hunk 8]
```diff
@@ -4,22 +4,22 @@
...
-      <p>
+      <div>
```
Comment (argilo) [inline src/var/www/hsmm-pi/View/NetworkServices/index.ctp]: I meant: change the `<span>` to a `<div>` and keep the `<p>`. `<div>` isn't allowed inside `<span>`.

Comment (urlgrey) [inline src/var/www/hsmm-pi/View/NetworkServices/index.ctp]: Ah, got it

**Comment by argilo** [comment]
@urlgrey I tested this branch on a Pi 3 and it's working well. You can probably go ahead and merge it. :+1:

**Comment by urlgrey** [comment]
Thanks @argilo , I'll squash the commits in a separate branch and merge to master.  Thanks again for all of the excellent feedback!

**Comment by urlgrey** [comment]
Rebased commits to `update-bootstrap-and-jquery` branch, will open separate PR

**Pull request closed without merging**